website/servers/servers.lib.php

```diff
@@ -15,3 +15,19 @@ function saveservers() {
 $PokemonServers = '.var_export($GLOBALS['PokemonServers'], true).';
 ');
 }
+
+
+$serverinfo = json_decode(
+	file_exists(__DIR__ . '/../../config/serverinfo.json') ?
+		file_get_contents(__DIR__ . '/../../config/serverinfo.json') :
+		"{}",
```

Comment:
```suggestion
	file_get_contents(__DIR__ . '/../../config/serverinfo.json') ?: "{}",
```

In general, any `exists` function is a race condition waiting to happen, and should not be used.